lib/aggregations/class-custom-date-range.php

```diff
@@ -18,6 +18,20 @@
  */
 class Custom_Date_Range extends Aggregation {
 
+	/**
+	 * The query var for the start date.
+	 *
+	 * @var string
+	 */
+	protected string $query_var_start = 'custom_date_range_start';
```

Comment:
nitpicky comment, but these are long query var names and it's generally a good idea to keep query vars short. browsers have url length limits which vary by browser (cool), and while they're usually pretty reasonable (~2k), you'd be surprised how quickly a search form can reach that limit. My suggestion would be to make these `date_gte` and `date_lte`.

Reply:
Thank you for that feedback, and that's a very good point! Naming stuff is hard 😝

Updated! 🙌